aspnetcore/diagnostics/asp0014.md

```diff
@@ -51,4 +51,17 @@ app.Run();
 
 ## When to suppress warnings
 
-Do ***not*** suppress a warning from this rule.
+Warnings from this rule can be supressed if the target `UseEndpoints` invocation is invoked without any mappings as a strategy to organize middleware ordering.
```

Comment:
```suggestion
Warnings from this rule can be suppressed if the target `UseEndpoints` doesn't impact middleware ordering, as shown in the following code:
```

Reply:
but `.UseEndpoints(_ => {})` is used to impact middleware ordering.

Reply:
> but `.UseEndpoints(_ => {})` is used to impact middleware ordering. see the referenced issue #28364

Thanks, reverted.